ivy_tests/test_ivy/test_frontends/test_paddle/test_fft.py

```diff
@@ -258,6 +260,40 @@ def test_paddle_ifftshift(
     )
 
 
+@handle_frontend_test(
+    fn_tree="paddle.fft.ihfft2",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_num_dims=2,
+        min_dim_size=2,
+    ),
+)
+def test_paddle_ihfft2(
+    dtype_x_axis,
+    norm,
+    frontend,
+    backend_fw,
+    test_flags,
+    fn_tree,
+):
+    input_dtypes, x, axis = dtype_x_axis
+    shape = ivy.array(x).shape
+    x = ivy.reshape(x, shape)  # reshape into a 2 dimensional array.
+
+    ivy.astype(
```

Comment:
We don't need to call `ivy` here only the `helpers.test_frontend_function` will run the tests